src/zc/relation/README.txt

```diff
@@ -107,9 +107,8 @@ name as the tokens.
     ...     # the next parts just make the tests prettier
     ...     def __repr__(self):
     ...         return '<Employee instance "' + self.name + '">'
-    ...     def __cmp__(self, other):
-    ...         # pukes if other doesn't have name
-    ...         return cmp(self.name, other.name)
+    ...     def __lt__(self, other):
+    ...         return self.name < other.name
```

Comment:
I don't like seeing only one of six rich comparison methods being defined. If you use @functools.total_ordering, you can get away by defining two of six.